plugins/ci/version.txt

```diff
@@ -1 +1 @@
-5.3.4
+5.3.5
```

Comment:
As this provides a new feature and also adds a new parameter (`customChecksDirectory`) support, It's worth considering using `5.4.0`

Reply:
done